.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-*	@sumeet-bansal
+* @sumeet-bansal
 * @shravanhariharan2
```

Comment:
another small nit: these are supposed to be on the same line (see [Github docs](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners)). I think that's why after rewrite you weren't getting auto-requested for reviews but I was